### PR TITLE
Fix #560  Added 4px margin between icons and tab labels in TheHeader

### LINKED ISSUE #560: [Feature]: Add 4px margin between icons and tab labels in `TheHeader`
### Feature Summary

`TheHeader` component has tabs that appear in the upper right, with icons and labels for each page e.g. 'Play'. The icons are too close to the text and would look better with a 4px margin between them. 

### Detailed Description

See the image below.
image

We should add 4px of margin between the icons and the text in the tabs inside `src/components/TheHeader.vue` so the elements have room to breathe.

### PR DESCRIPTION
**Pull request by dushyantpant5**
## Issue number #560 

Relevant [issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #

## Please check the following

-  All tests ran successfully
- The code is formatted properly , according to the guidelines
- For New Features:
  - No tests added
  - No documentation update

Screenshot 2023-09-23 at 7 55 43 PM

### CODE REVIEW

File: src/components/TheHeader.vue [hunk 1]
```diff
@@ -23,6 +23,7 @@
             :data-cy="cyName"
             :title="text"
             :to="page"
+            :style="{margin :'4px'}"
```
Comment (itsalaidbacklife) [inline src/components/TheHeader.vue:26]: The intent of the issue is to add a margin between the icon and text, not between the tabs themselves. That means we should move this onto the icon itself and specifically add margin-right. 

Additionally, we normally favor using classes that utilize preexisting Vuetify styles ((docs:)[https://vuetifyjs.com/en/styles/spacing/]), in this case for a right margin of 4px (a single unit of Vuetify's spacing helpers), that would be 
```class="mr-1"```

File: src/components/TheHeader.vue [hunk 2]
```diff
@@ -23,6 +23,7 @@
...
           >
             <v-icon
               :icon="`mdi-${icon}`"
```
Comment (itsalaidbacklife) [inline src/components/TheHeader.vue:29]: This is where we should put the margin class so that it goes between the icon and text

**Review by itsalaidbacklife — COMMENTED** [review]
Thanks for your contribution. This is the right idea, though there is some fine tuning needed for it to be merged. Details are in the comments. If you have any questions, feel free to ask them in the comments

File: src/components/TheHeader.vue (hunk 2 above)
Comment (dushyantpant5) [reply src/components/TheHeader.vue:29]: I opened a new PR with required changes. Please check

**Pull request closed without merging**